packages/demo/usecase-demo/src/modules/DisclaimerModal.tsx

```diff
@@ -34,32 +33,36 @@ export function DevModal() {
   useEffect(() => {
     setStorageDevMode(isDevMode);
   }, [isDevMode]);
-  
+
   useEffect(() => {
     if (open == false) {
-      setStorageModalViewed(true);
+      setStorageDiscaimerViewed(true);
     }
   }, [open]);
 
-  if (!isStorageModalViewed) {
+  if (!isStorageDiscaimerViewed) {
```

Comment:
We will avoid putting too much code in a big `if` like this.
We prefer to **break early**.

Example here:

if (isStorageDiscaimerViewed) {
  return false;
}

return (
  <Dialog ...

Reply:
done